dream_layer_backend/img2img_workflow.py

```
@@ -74,30 +77,32 @@ def transform_to_img2img_workflow(data):
     batch_size = max(1, min(8, int(data.get('batch_size', 1))))
     steps = max(1, min(150, int(data.get('steps', 20))))
     cfg_scale = max(1.0, min(20.0, float(data.get('cfg_scale', 7.0))))
-    denoising_strength = max(0.0, min(1.0, float(data.get('denoising_strength', 0.75))))
+    denoising_strength = max(
```

Ruff (0.11.9)

Local variable `denoising_strength` is assigned to but never used

Remove assignment to unused variable `denoising_strength`

(F841)